### PR TITLE
create only private aks subnets

### PR DESCRIPTION
**Pull request by dutsmiller**

### CODE REVIEW

File: output.tf [hunk 1]
```diff
@@ -52,7 +52,7 @@ output "aks" {
   description = "Virtual network information matching AKS module input."
   value = { for aks_id,info in local.aks_info:
     aks_id => {
-      subnets = {
+      cidrs = {
         private = {
```
Comment (jhisc) [inline output.tf:56]: Should this just move up a level?

File: variables.tf [hunk 2]
```diff
@@ -62,7 +62,6 @@ variable "aks_subnets" {
   description = "AKS subnets"
   type = map(object({
     private = any
```
Comment (jhisc) [inline variables.tf:64]: Same here?

File: locals.tf [hunk 3]
```diff
@@ -20,7 +20,6 @@ locals {
   aks_subnets = merge([for id in keys(local.aks_info): 
     {
       "aks-${id}-private" = merge({aks_id = id}, merge(var.subnet_defaults, local.aks_info[id].private) )
```
Comment (jhisc) [inline locals.tf:22]: drop the -private ?

**Review by jhisc — CHANGES_REQUESTED** [review]

**Review by jhisc — APPROVED** [review]

**Comment by fabiendelpierre** [comment]
@dutsmiller I guess this means we'll have to update the naming repo here: https://github.com/LexisNexis-RBA/terraform-azurerm-naming/blob/7a866a170fad7178c22f6f48e11ee609b2d1e42e/custom.json#L162-L163

**Pull request merged**